lucien-ios/Lending/LendABookViewController.swift

```diff
@@ -0,0 +1,153 @@
+//
+//  LendABookViewController.swift
+//  lucien-ios
+//
+//  Created by Ahmed, Guled on 10/19/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import UIKit
+import RxSwift
+import RxCocoa
+
+class LendABookViewController: UIViewController, UITableViewDelegate, AlertDisplaying {
+
+    // MARK: - Private IBOutlets
+
+    @IBOutlet private weak var comicPhoto: UIImageView!
+    @IBOutlet private weak var seriesTitleLabel: UILabel!
+    @IBOutlet private weak var storyTitleLabel: UILabel!
+    @IBOutlet private weak var volumeLabel: UILabel!
+    @IBOutlet private weak var tableView: UITableView!
+    @IBOutlet private weak var comicInfoView: UIView!
+
+    // MARK: - Private Constants
+
+    private let disposeBag = DisposeBag()
+    private let viewModel = LendingBookViewModel()
+
+    // MARK: - Private Variables
+
+    private var nextButton = UIBarButtonItem()
+    private let lendingCell = UINib(nibName: "LendingTableViewCell", bundle: nil)
+    private let headerView = UIView()
+
+    // MARK: - Variables
+
+    var selectedIndex: Int?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureNavigationController()
+        configureTableView()
+
+        viewModel.getAllUsers { [weak self] (users, error) in
+            if error == nil {
+                guard let users = users else { return }
+                self?.viewModel.users.value = users
```

Comment:
This is a strange pattern to me. Why wouldn't the viewmodel just set this on it's own?